### PR TITLE
Nodred

### PR DESCRIPTION
**Pull request by wenwenchenbosch**

### CODE REVIEW

File: examples/node-red/README.md [hunk 1]
```diff
@@ -5,12 +5,13 @@ After installing [node-red](https://nodered.org/) and [node-red-dashboard](https
 ## MQTT
```
Comment (SebastianSchildt) [inline examples/node-red/README.md:5]: Also needs something called node-red-contrib-web-worldmap , maybe mention it

File: examples/node-red/README.md [hunk 2]
```diff
@@ -5,12 +5,13 @@ After installing [node-red](https://nodered.org/) and [node-red-dashboard](https
...
 *Note*: Do not forget to config your kuksa-val-server to publish the needed topics using the option `--mqtt.publish`.
 
 ## Websocket
 - [websocket-subscription.json](./websocket-subscription.json) do the same like the mqtt example above via websocket subscription feature.
 - [websocket-advanced.json](./websocket-advanced.json) implements a test client and uses secure connection with server
 
+![screenshot](./node-red-screenshot.png)
+
 ### TODO
 It is also possible to use node-red as websocket client, it is also configured in the json file, but it does not work jet
```
Comment (SebastianSchildt) [inline examples/node-red/README.md:17]: This is possible now

**Review by SebastianSchildt — CHANGES_REQUESTED** [review]
Tested, and all works. Just adapt the Readme (dependency and TODO note) and this is fine.

Not a problem, but I noticed, the websocket examples uses Vehicle.OBD.Speed and MQTT uses Vehicle.Speed

**Comment by wenwenchenbosch** [comment]
> Tested, and all works. Just adapt the Readme (dependency and TODO note) and this is fine.
> 
> Not a problem, but I noticed, the websocket examples uses Vehicle.OBD.Speed and MQTT uses Vehicle.Speed

Hi @SebastianSchildt: good reviewed :+1:  But which one should I use?

**Comment by wenwenchenbosch** [comment]
> > Tested, and all works. Just adapt the Readme (dependency and TODO note) and this is fine.
> > Not a problem, but I noticed, the websocket examples uses Vehicle.OBD.Speed and MQTT uses Vehicle.Speed
> 
Hi @SebastianSchildt: all review findings fixed

**Review by SebastianSchildt — APPROVED** [review]
Only changes Readme, and extends/replaces the nodered flows already in master

Can be merged 👯‍♀️    @daschubert @eriksven

**Pull request merged**